astro/src/pages/services/evaluations/voices.astro

```diff
@@ -0,0 +1,100 @@
+      </h1>
+    </div>
+    <div class="py-3">
+      <p class="text-bg-hero">
+        <Branding>Accessible Community</Branding> provides digital accessibility
+        evaluations and remediation services for inclusion-based nonprofits.
+      </p>
+    </div>
+  </Hero>
+
+  <ThemedSection style="tertiary">
+    <h2 class="display-4">
+      Are you <strong>supporting</strong> people with disabilities?
+    </h2>
+    <p>
+      At <Branding>Accessible Community</Branding>, we believe that we must each ask ourselves, “What can I do to reach out to and support individuals hurt and isolated by the recent rollback in accessibility and inclusion in the United States?” Coming together and supporting each other is an important step to improving our country and world. Everyone affected should have a voice in creating solutions. Accessible Community wants to make sure people with disabilities can participate in the conversation and the path forward.
+    </p>
+    <p>
+      To that end, <Branding>Accessible Community</Branding> will be focusing on providing accessibility evaluation and remediation services to inclusion-based nonprofits in U.S. communities.
+    </p>
+
+    <h3>What Accessible Community will do.</h3>
+    <ol>
+      <li>Select smaller nonprofits working towards creating more inclusive communities,</li>
+      <li>Raise funds to pay our evaluators to make sure their websites are accessible,</li> 
+      <li>Provide an evaluation against WCAG 2.1 A & AA as well as accessibility best practices that improve reach and impact,</li>
+      <li>Provide technical assistance to make the necessary improvements to the site, and</li>
+      <li>Advertise nonprofits whose sites are accessible to people with disabilities.</li>
+    </ol>
+
+    <h3>What you can do.</h3>
+    <ol>
+      <li>Recommend nonprofits that promote inclusion in your community (feel free to nominate a nonprofit you run),</li>
+      <li>Vote on which nonprofit we should assist next,</li>
+      <li>Donate to support these efforts.</li>
+    </ol>
+
+    <h3>Your <strong>impact</strong>.</h3>
+    <p>
+      Each remediation costs $2,000. We will start a remediation for a nonprofit as soon as we reach that amount. The majority of the funds go to experts between jobs in digital accessibility or emerging experts learning honing their accessibility skillset.
+    <p>
+      Your donation promotes disability-inclusion in three ways:
+    </p>
+    <ol>
+      <li>Helping nonprofits that promote inclusion reach a wider audience,</li>
+      <li>Supporting accessibility professionals impacted by the loss of inclusion programs, and</li>
+      <li>Providing emerging accessibility professionals with learning opportunities.</li>
+    </ol>
+    <p>
+      If you run an organization and would like <Branding>Accessible Community</Branding> to help you improve your reach, please visit our evaluation services page and sign up for an evaluation.
```

Comment:
This initiative will focus on inclusion-centered non-profits with 10 or fewer paid employees. If you run an organization that doesn't fit in this category and and would like...